doc/classes/VisualServer.xml

```diff
@@ -3157,6 +3157,8 @@
 			</argument>
 			<argument index="2" name="scale" type="bool">
 			</argument>
+			<argument index="3" name="use_filter" type="bool">
```

Comment:
Should the parameter be called `use_filter` or `filter`?

Reply:
On the font resources, it's called `use_filter` so I'd go with this one.

Reply:
Could you also mention what `[code]use_filter[/code] enables in the description?